---
title: "'Throw' 陳述式不可以在 'Catch' 陳述式外或 'Finally' 陳述式內省略運算元"
ms.date: 07/20/2015
f1_keywords:
- vbc30666
- bc30666
helpviewer_keywords:
- BC30666
ms.assetid: a208a6ea-0e36-4bf1-8984-4de1a0e38a2a
ms.openlocfilehash: 6906cb743e6f36acfbf51a5a38d03bd05fde98e2
ms.sourcegitcommit: 6b308cf6d627d78ee36dbbae8972a310ac7fd6c8
ms.translationtype: MT
ms.contentlocale: zh-TW
ms.lasthandoff: 01/23/2019
ms.locfileid: "54598759"
---
# <a name="throw-statement-cannot-omit-operand-outside-a-catch-statement-or-inside-a-finally-statement"></a>'Throw' 陳述式不可以在 'Catch' 陳述式外或 'Finally' 陳述式內省略運算元
`Throw` 陳述式外的 `Catch` 陳述式必須提供例外狀況物件的名稱。  
  
 **錯誤 ID:** BC30666  
  
## <a name="to-correct-this-error"></a>更正這個錯誤  
  
1.  指定衍生自 `System.Exception`之例外狀況物件的名稱。  
  
2.  重建您的程式碼，讓 `Throw` 陳述式位於 `Catch` 區塊內。  
  
## <a name="see-also"></a>另請參閱
- [Throw 陳述式](../../visual-basic/language-reference/statements/throw-statement.md)
- [Try...Catch...Finally 陳述式](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
- [在 Visual Basic 中的例外狀況類別](https://msdn.microsoft.com/library/9aac396f-34ca-4afb-8e6c-e523cb690ba9)
- [Visual Basic 中的例外狀況和錯誤處理](https://msdn.microsoft.com/library/3e351e73-cf23-40ab-8b60-05794160529e)
